### PR TITLE
Death weapon balance

### PR DESCRIPTION
**Pull request by shalkya**
improve DeathWeapon of experimental and rebalance them

### CODE REVIEW

**Comment by KionX** [comment]
`position[3] = position[3]+5`

How is unit rotation taken into account here?

**Comment by shalkya** [comment]
nicely spotted thx. Please test that to be sure it's working as intended.

**Comment by KionX** [comment]
Checked the colossus. Works correctly.
Verification Method: DrawCircle (position, 3, '0000ff')

It's not needed
local current_yaw = math.floor (a * (180 / math.pi) + 0.5)
Enough
position [3] = position [3] + 5 * math.cos(a)
position [1] = position [1] + 5 * math.sin(a)

File: units/URL0402/URL0402_unit.bp [hunk 1]
```diff
@@ -781,11 +781,11 @@ UnitBlueprint {
         {
             Damage = 4000,
             DamageFriendly = true,
-            DamageRadius = 7,
+            DamageRadius = 6,
             DamageType = 'DeathExplosion',
             DisplayName = 'Death Weapon',
             DummyWeapon = true,
-            Label = 'DeathWeapon',
+            Label = 'SpiderDeath',
```
Comment (Crotalus) [inline units/URL0402/URL0402_unit.bp]: Make sure this rename doesn't cause side effects, if I remember correctly there is some checks if label == 'DeathWeapon' in lua/sim/Unit.lua

Comment (shalkya) [reply units/URL0402/URL0402_unit.bp]: will check

Comment (shalkya) [reply units/URL0402/URL0402_unit.bp]: looked into it, it shouldn't cause any issue.

**Pull request merged**